docs/usage.rst

```diff
@@ -544,6 +552,100 @@ and the ``!!python/object`` tag has disappeared.
     d_yaml = d.to_yaml_s()
     print(d_yaml)
 
+Resolve file paths
+------------------
+
+Portability is a core feature of
+:mod:`audobject`.
+Now assume we have an object
+that takes as argument the path to a file.
+When we serialize the object
+we want to make sure that:
+
+1. we store the file path relative to the YAML file
+2. the path is correctly expanded when we re-instantiate the object
+
+This can be achieved using
+:class:`audobject.FilePathResolver`.
+
+.. jupyter-execute::
+
+    class MyObjectWithFile(audobject.Object):
+
+        @audobject.init_decorator(
+            resolvers={
+                'path': audobject.FilePathResolver,  # ensure portability
+            }
+        )
+        def __init__(
+                self,
+                path: str,
+        ):
+            self.path = path
+
+        def read(self):  # print path and content
+            print(self.path)
+            with open(self.path, 'r') as fp:
+                print(fp.readlines())
+
+Here, we create a file and pass it to the object.
+
+.. jupyter-execute::
+
+    import os
+    import audeer
+
+    root = 'root'
+
+    res_path = os.path.join(root, 're', 'source.txt')  # root/re/source.txt
+    audeer.mkdir(os.path.dirname(res_path))
+    with open(res_path, 'w') as fp:
+        fp.write('You found me!')
+
+    o = MyObjectWithFile(res_path)
+    o.read()
+
+But when we serialize the object,
+we can see that the path is
+stored relative to the directory
+of the YAML file.
+
+.. jupyter-execute::
+
+    import yaml
+
+
+    yaml_path = os.path.join(root, 'yaml', 'object.yaml')  # root/yaml/object.yaml
+    o.to_yaml(yaml_path)
+
+    with open(yaml_path, 'r') as fp:
+        content = yaml.load(fp, Loader=yaml.Loader)
+    content
+
+When we re-instantiate the object
+the path gets expanded again.
+
+.. jupyter-execute::
+
+    o2 = audobject.Object.from_yaml(yaml_path)
+    o2.read()
+
+This will also work from another location
+(note that we also move the referenced file here,
+as its relative location to the YAML file must not change).
```

Comment:
```suggestion
This will also work from another location.
Note that we have to move all referenced files as well,
as their relative location to the YAML file must not change.
```